app/src/main/java/de/westnordost/streetcomplete/data/StreetCompleteSQLiteOpenHelper.kt

```diff
@@ -114,7 +114,7 @@ import de.westnordost.streetcomplete.quests.oneway_suspects.data.WayTrafficFlowT
                 INSERT INTO ${VisibleQuestTypeTable.NAME} (
                     ${VisibleQuestTypeTable.Columns.QUEST_PRESET_ID},
                     ${VisibleQuestTypeTable.Columns.QUEST_TYPE},
-                    ${VisibleQuestTypeTable.Columns.VISIBILITY},
+                    ${VisibleQuestTypeTable.Columns.VISIBILITY}
                 ) SELECT
```

Comment:
> Insert here rant about any system not allowing trailing commas or demanding to not include them. It results in uglier diffs and completely pointless bugs like this one. For no benefit at all.

Note that you can at least improve the situation using leading comma formatting, although it seems to mostly/only be popular in functional programming circles.

```
                INSERT INTO ${VisibleQuestTypeTable.NAME}
                    ( ${VisibleQuestTypeTable.Columns.QUEST_PRESET_ID}
                    , ${VisibleQuestTypeTable.Columns.QUEST_TYPE}
                    , ${VisibleQuestTypeTable.Columns.VISIBILITY}
                    )
                SELECT
```
Admittedly it doesn't work outside of parentheses, like in the SELECT below.

Reply:
Given it's just building a query, and they're all constants so the raw query is useless on it's own, what about joining a set with commas, which would be safe from all of that and self-validating via the compiler?